openslides_backend/shared/interfaces.py

```diff
@@ -47,7 +48,7 @@ class Services(Protocol):  # pragma: no cover
 
     # TODO: Use correct type here. Fitting together dependency_injector and our
```

Comment:
Can the todo be removed?

Reply:
I guess, this is a todo for the other two lines under authentication (they're set as `any`, too).